.phpstan/ForbiddenMethodCallsRule.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\PHPStan;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\FuncCall;
+use PhpParser\Node\Name;
+use PHPStan\Analyser\Scope;
+use PHPStan\Rules\Rule;
+use PHPStan\Rules\RuleError;
+use PHPStan\Rules\RuleErrorBuilder;
+
+/**
+ * PHPStan rule that forbids usage of specific debugging and output functions.
+ *
+ * This rule enforces code quality by preventing debugging functions from being
+ * committed to the codebase, helping maintain clean production code.
+ *
+ * @implements Rule<FuncCall>
+ * @author Claude <noreply@anthropic.com>
+ */
+final class ForbiddenMethodCallsRule implements Rule
```

Comment:
You could consider using https://github.com/spaze/phpstan-disallowed-calls for this.